Gemfile

```diff
@@ -36,13 +36,15 @@ gem "bootsnap", require: false
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri windows ], require: "debug/prelude"
+  gem "debug", "~> 1.10.0", platforms: %i[mri windows], require: "debug/prelude"
 
   # Static analysis for security vulnerabilities [https://brakemanscanner.org/]
-  gem "brakeman", require: false
+  gem "brakeman", "~> 7.0"
```

Comment:
```suggestion
  gem "brakeman", require: false
```

As per above comment, and this should still have require false. We use require false for tooling not needed as part of running the app and brakeman is only used as part of CI